actions/templates.go

```diff
@@ -258,38 +256,29 @@ func PutTemplate(c *gin.Context) {
 func DeleteTemplate(c *gin.Context) {
 	u := middleware.GetUser(c)
 
-	keys, err := storage.GetSesKeys(c, u.ID)
+	id, err := strconv.ParseInt(c.Param("id"), 10, 64)
 	if err != nil {
 		c.JSON(http.StatusBadRequest, gin.H{
-			"message": "AWS Ses keys not set.",
+			"message": "Id must be an integer",
 		})
 		return
 	}
 
-	store, err := templates.NewSesTemplateStore(keys.AccessKey, keys.SecretKey, keys.Region)
+	template, err := storage.GetTemplate(c, id, u.ID)
```

Comment:
Here we use storage, but below we use the service. Can we do them both with the service methods?

Reply:
Yeah it makes sense